src/services/coreBackService.js

```diff
@@ -28,10 +28,10 @@ function generateAxiosConfig(req) {
   };
 }
 
-function postJourneyEvent(req, event) {
+function postJourneyEvent(req, event, queryParams = {}) {
```

Comment:
It feels a bit odd to pass arbitrary query params here - I think this method should capture the details of the API interface, which I think includes the specific parameters we expect (i.e. `currentPage`).